lib/engine/game/g_18_eus/round/bidbox_auction.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module Engine
+  module Game
+    module G18EUS
+      module Round
+        module BidboxAuction
+          attr_accessor :bids
+
+          def setup
+            @game.setup_bidboxes
```

Comment:
nit: could this be called something like `@game.next_auctions!`?

Reply:
Renamed